index.js

```diff
@@ -37,16 +40,19 @@ const config = module.exports.config = async (c = {}) => {
       'expressjs/serve-static',
       'expressjs/errorhandler',
       'expressjs/serve-index',
+      'expressjs/security-wg',
```

Comment:
I am on the fence for these ones. The original idea was to only have ones which are published and active. How does the badge board look for this? Same for the `.github` ones.

Reply:
The security-wg repository would be good to include, like the discussion one, since that place will have issues that could be of interest to the collaborators

I will delete the .github ones

Reply:
Yeah, I wonder if we need a flag for which should go on the badge board so we dont have empty rows for the "discussion" type repos?

Reply:
This would be a good option to add to the package in pkgs